cmd/sync.go

```diff
@@ -128,11 +128,12 @@ func (raw *rawSyncCmdArgs) parseMaxObjectIndexerMapInGB() (uint32, error) {
 
 	value, err := strconv.Atoi(raw.maxObjectIndexerMapSizeInGB)
 	if err != nil {
-		return uint32(value), fmt.Errorf("Parsing failed for maxObjectIndexerMapSizeInGB with error: %v", err)
+		err = fmt.Errorf("Parsing failed for maxObjectIndexerMapSizeInGB (%s) with error: %v", raw.maxObjectIndexerMapSizeInGB, err)
+		return uint32(value), err
```

Comment:
nit, pls do return 0 instead of uint32(value), since accessing value on err is not technically correct

Reply:
No Tomar i checked the implementation of strconv.atoi for error case it return the value 0 only. So that's why i didn't change it.